docs/docs/launch-manual/sections/core-hacking/sections/creating-a-fork-of-a-core-package.md

```diff
@@ -0,0 +1,93 @@
+## Creating a Fork of a Core Package
+
+Several of Pulsar's core packages are maintained in the
+[`packages` directory of the pulsar-edit/pulsar repository](https://github.com/pulsar-edit/pulsar/tree/master/packages).
+If you would like to use one of these packages as a starting point for your own
+package, please follow the steps below.
+
+::: tip Tip
+
+In most cases, we recommend [generating a brand new package](#package-generator)
+or a [brand new theme](#creating-a-syntax-theme) as the starting point for your
+creation. The guide below applies only to situations where you want to create a
+package that closely resembles a core Pulsar package.
+
+:::
+
+### Creating Your New Package
+
+<!--Could this be made better with GH CLI?-->
```

Comment:
Probably yes because I think we can avoid the whole clunky "download a zip and copy it" stuff. Honestly I'm not that well versed in GH CLI so I would need to check. As it is new content I'd be happy to put this one in first and make a new PR for the GH CLI instructions as it will be seen by more people for comment if I make a stupid mistake.

Reply:
Yeah that's not a bad idea. Leave as is for now, then if we can do tabbed instructions that'd be nice. As we could probably do it with `git` and `gh CLI` and who knows what else as people want to add it there. So I'm happy calling this resolved for now and we can modify it later.

Reply:
Added to the ever growing list